OMEdit/OMEditLIB/MainWindow.cpp

```diff
@@ -1170,6 +1170,14 @@ void MainWindow::checkAllModels(LibraryTreeItem *pLibraryTreeItem)
 
 void MainWindow::exportModelFMU(LibraryTreeItem *pLibraryTreeItem)
 {
+  // check for supported targetLanguage C or Cpp
+  QString targetLanguage = OptionsDialog::instance()->getSimulationPage()->getTargetLanguageComboBox()->currentText();
+  if (targetLanguage.compare("C") != 0 && targetLanguage.compare("Cpp") != 0) {
+    MessagesWidget::instance()->addGUIMessage(MessageItem(MessageItem::Modelica, QString("Target Language <b>%1</b> is not supported for FMU Export. Only <b>C</b> and <b> CPP </b> are supported").arg(targetLanguage),
+                                                                  "FMU_EXPORT Failed", Helper::errorLevel));
```

Comment:
There are extra spaces around `CPP`.

Any string that is supposed to be shown in the UI should be used with `tr` so it is marked for translation.

Reply:
fixed with https://github.com/OpenModelica/OpenModelica/pull/13142/commits/79395e5b718fcd2262996f8df9dc5b1fc1783abb